### PR TITLE
First pull request, clone of trials project

### PR DESCRIPTION
**Pull request by ReshyResh**
# Trials Project Review
- Created new repo.
- Cloned the Microverse trials project.
- Added linters for code check-ups.

### CODE REVIEW

**Comment by ReshyResh** [comment]
# Required Changes ♻️

- Fix linter errors 

# Optional suggestions

Use semantic tags instead of opening many divs.

Cheers and Happy coding!👏👏👏

**Comment by ReshyResh** [comment]
# Required Changes ♻️

   - Change background color to a light blue

**Comment by ReshyResh** [comment]
# Congratulations! 🎉
Your project is complete! There is nothing else to say other than... it's time to merge it 🛳

Happy coding. Cheers! 🚀🚀

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  font-size: 40px;
+  padding-top: 3%;
+  font-weight: bold;
+  color: #3c4047;
+  text-align: center;
+}
+
+#title-div {
+  background-color: #d1e7f0;
+}
```
Comment (Temmarie) [inline css/style.css:17]: - [ ] A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary like this current review. :wink:
This is an example inline comment. :slightly_smiling_face:
You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:
To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

- [ ] Also, be sure to make polite suggestions when giving a review adding a reason why the change is necessary as well.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @ReshyResh    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You added a summary at the end of the review ✔️ 

 However, there are changes required for you to make for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Required Changes** ♻️

- [ ] I think it would have been great if you use did use the GitHub inline comments feature at least once to leave review comments. For this project, kindly leave the comment under a line or block of code that fits the context the comment is about. This not only makes the comment complete but understandable as well. I will try to invite you to check this [lesson on how to add inline comments](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request).


_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  max-width: 40%;
+  min-width: 30%;
+}
+
+li {
+  display: inline-block;
+  text-align: justify;
+  margin: 20px;
+  float: left;
+}
```
Comment (ReshyResh) [inline css/style.css:63]: # Required changes ♻️
- Add a text-justify rule for better readability.

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @ReshyResh  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**